.gitignore

```diff
@@ -1,5 +1,5 @@
 *.hdf5
-
+*.ipynb
```

Comment:
Generally a good idea to exclude notebook files from a repo like you are doing here. In case you want to include a notebook in future, just be sure to only add _unrendered_ notebooks, i.e., first do "restart kernel and clear all cells" and resave the notebook before doing `git add`. An unrendered notebook only occupies a small file size, where as a rendered one be pretty large, so this helps minimize the size of a source-code repo.